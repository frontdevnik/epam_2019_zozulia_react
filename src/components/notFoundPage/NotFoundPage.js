import React from 'react';
import styles from './notFoundPage.module.scss';

function NotFoundPage() {
  return (
    <div className={styles.main}>
      <h1 className={styles.text}>Page not found</h1>
    </div>
  );
}

export default NotFoundPage;
