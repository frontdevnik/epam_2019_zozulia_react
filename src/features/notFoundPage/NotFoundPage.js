import React from 'react';
import NotFoundPage from '../../components/notFoundPage/NotFoundPage';

function notFoundPage() {
  return (
    <NotFoundPage />
  );
}

export default notFoundPage;
