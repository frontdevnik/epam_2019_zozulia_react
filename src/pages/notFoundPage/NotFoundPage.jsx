import React from 'react';

import NotFoundPage from '../../components/notFoundPage/NotFoundPage';
import Header from '../../components/header';

export default () => (
  <>
    <Header />
    <NotFoundPage />
  </>
);
