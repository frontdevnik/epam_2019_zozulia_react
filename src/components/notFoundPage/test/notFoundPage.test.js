import React from 'react';
import NotFoundPage from "../NotFoundPage";
import styles from "../notFoundPage.module.scss";

describe('<NotFoundPage />', () => {
    let component;

    beforeEach(() => {
        component = shallow(<NotFoundPage />);
    })

    it('should render correctly', () => {
        expect(component.html()).toMatchSnapshot()
    })

    it('should has main class', () => {
        expect(component.find('div').hasClass(styles.main)).toBeTruthy()
    })

    it('should text has correct class', () => {
        expect(component.find('h1').hasClass(styles.text)).toBeTruthy()
    })
})
