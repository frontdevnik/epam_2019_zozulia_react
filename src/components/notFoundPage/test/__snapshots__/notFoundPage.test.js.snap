// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render correctly 1`] = `"<div class=\\"main\\"><h1 class=\\"text\\">Page not found</h1></div>"`;

exports[`<NotFoundPage /> should text has correct class 1`] = `"<div class=\\"main\\"><h1 class=\\"text\\">Page not found</h1></div>"`;
